extensions/amp-form/0.1/amp-form.js

```diff
@@ -335,6 +339,17 @@ export class AmpForm {
     }
   }
 
+  handleClearAction_() {
+    this.form_.reset();
+    this.setState_(FormState_.INITIAL);
+    this.form_.classList.remove('user-valid', 'user-invalid');
```

Comment:
`remove` only takes a single argument.

Reply:
according to following doc, `remove` takes multiple arguements
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

Reply:
See the Browser Compatibility table at the bottom, "Multiple arguments for add() & remove()". We're limited by IE/Edge support here.